package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "node --experimental-modules ./scripts/src2dist.js"
+    "build": "node --experimental-modules scripts/builder.js"
```

Comment:
It does not use `alpheios-node-build` at all because it does not build any code. This line runs the script that converts data from the source directory to a dist one.